azure-pipelines.yml

```diff
@@ -467,6 +467,35 @@ jobs:
     parameters:
       includePatterns:
         - "/hw/***"
+  - task: DownloadSecureFile@1
+    name: gcpkey
+    inputs:
+      secureFile: gcpkey.json
+  - bash: |
+      echo "Installing gsutil"
+      sudo apt-get install -y apt-transport-https ca-certificates gnupg
+      echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" \
+        | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+      curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo tee /usr/share/keyrings/cloud.google.gpg
+      sudo apt-get update || {
+        error "Failed to run apt-get update"
+      }
+      sudo apt-get install -y google-cloud-cli
```

Comment:
Should this be in ci/install-package-dependencies.sh instead?

Reply:
I kept it out of install-package-dependencies.sh since `gsutil` is only needed in this one place, and I didn't want to spend time installing stuff we don't need on every agent that the pipeline touches. If GCP ends up being used more widely in the pipeline, then it should be pretty simple to move this into the install script.